src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -26,100 +27,126 @@ import { actionHandler } from "../common/directives/action-handler-directive";
 import { hasAction } from "../common/has-action";
 import { ActionHandlerEvent } from "../../../data/lovelace";
 import { handleAction } from "../common/handle-action";
+import { subscribeRenderTemplate } from "../../../data/ws-templates";
 
 class HuiGenericEntityRow extends LitElement {
-  @property() public hass?: HomeAssistant;
+  @property() public showSecondary: boolean = true;
+  @property() private _hass?: HomeAssistant;
+  @property() private _config?: EntitiesCardEntityConfig;
+  @property() private _content?: string = "";
+  @property() private _unsubRenderTemplate?: Promise<UnsubscribeFunc>;
 
-  @property() public config?: EntitiesCardEntityConfig;
+  public set config(config: EntitiesCardEntityConfig) {
+    this._config = config;
+    this._disconnect().then(() => {
```

Comment:
Why do we want to reset the connection when config is changed?

Reply:
I just copied from markdown-card :rofl:

Reply:
The reason is that the template - which in the case of markdown-card is almost always changed if the configuration is - sent in the subscription request. So a new template requires a new subscription.

Reply:
It could be done much more intelligently, starting with only connecting at all iff `secondary_info: markdown`.